src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    public Bulldozer(String nameMachine) {
+        super(nameMachine);
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println(getNameMachine() + " start work");
```

Comment:
The message should be more informative and indicate that the Bulldozer is starting to work. Consider specifying the type of machine in the message, for example: 'Bulldozer ' + getNameMachine() + ' started working.'

Reply:
can you apply my hm ?